src/cookieSyncManager.ts

```diff
@@ -159,4 +204,12 @@ export const isLastSyncDateExpired = (
         new Date().getTime() >
         new Date(lastSyncDate).getTime() + frequencyCap * DAYS_IN_MILLISECONDS
     );
-};
\ No newline at end of file
+};
+
+export const isTcfApiAvailable = (): boolean => {
+    if (typeof window.__tcfapi === 'function') {
+        return true;
+    } else {
+        return false;
+    }
+}
```

Comment:
I think we may have a util that checks if something is a function (I could be wrong). But if we do, you can probably make this a slick one-liner. 😎 

```suggestion
export const isTcfApiAvailable = (): boolean =>  isFunction(window.__tcfapi)';
```